src/mlt++/mlt++.vers

```diff
@@ -717,3 +717,11 @@ MLT_7.6.0 {
       "Mlt::Properties::copy(Mlt::Properties&, char const*)";
     };
 } MLT_7.4.0;
+
+MLT_7.10.0 {
```

Comment:
Change this symbols block name to `MLT_7.12.0`

Reply:
Done